mecab/python/setup.py

```
@@ -9,7 +9,14 @@ def cmd1(command):
 def cmd2(command):
     return cmd1(command).split()
 
-setup(name = "mecab-python",
+# Windows requires special prep
+if os.name == 'nt':
```

Ruff (0.8.2)

Undefined name `os`

(F821)